lib/constants.ts

```diff
@@ -1,3 +1,4 @@
 const BASE_URL = process.env.PRODURL ?? 'http://localhost:3000';
+const PAGE_SIZES: string[] = ['base', 'sm'];
```

Comment:
Would a better name for this be something like MOBILE_PAGE_SIZES? PAGE_SIZES seems to abstract.

Reply:
I think it's okay to be PAGE_SIZES because we may have to make changes for tablets or extremely large screens so I kept it generic for now